packages/shared_preferences/shared_preferences/pubspec.yaml

```diff
@@ -12,12 +12,21 @@ flutter:
         pluginClass: SharedPreferencesPlugin
       ios:
         pluginClass: FLTSharedPreferencesPlugin
+      macos:
+        default_package: shared_preferences_macos
+        pluginClass: SharedPreferencesPlugin
```

Comment:
I believe we shouldn't have pluginClass here

Reply:
Done